### ISSUE TITLE
Are tiles normalized by Macenko?

### ISSUE THREAD

**Issue by bswhite** [KatherLab/marugoto#12]
Hello,

I am interested in applying your pre-trained model for MSI prediction to an external dataset. Thank you making it available. How do you recommend normalizing my dataset?

The top-level README refers to Macenko normalization. Similarly, the Wagner et al bioRxiv manuscript says that tiles are normalized by applying Macenko to the tile in Fig A.1. 

Is this Macenko normalization implicitly done by the tile extraction code (normal_transform in marugoto/extract/extract.py)? If not, do you recommend modifying the code to perform Macenko normalization? And could you provide Fig A.1 in raw format?

Thank you,
Brian

**Comment by bswhite** [comment]
Hello again,

In searching a little more, I suspect the normalization in extract.py is _not_ Macenko performed against Fig A.1, because:
1. Macenko operates in OD space, whereas this is a transformation of the mean and std dev in RGB.
2. It appears the mean and std dev used are from ImageNet.

If the above is correct, how and when during the normalization process do you recommend performing Macenko so as to be consistent with the Wagner et al. report? (And, once again, would you mind providing Fig A.1?)

Thank you!
Brian

**Comment by Avic3nna** [comment]
Hi Brian @bswhite,

You can find our latest protocol + accompanying documentation and manual in the following link: 
https://github.com/KatherLab/STAMP

The paper will answer most of the questions you might have. It is based on the implementation of Wagner et al. New issues can be opened there, if you'd like. Thank you!
